metcdv3/balancer.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	"github.com/lytics/metafora"
 
-	etcdv3 "go.etcd.io/etcd/clientv3"
+	etcdv3 "go.etcd.io/etcd/client/v3"
```

Comment:
[Q] How would we refer to this package if we weren't using the alias? `client`? That might be cleaner?

Reply:
it turns into clientv3

Reply:
etcdv3 is easier to read imo since we don't use the server